### PR TITLE
nginx and ssl/tls updates.

### PR DESCRIPTION
**Pull request by cwlinode**

### CODE REVIEW

File: docs/web-servers/nginx/enable-tls-on-nginx-for-https-connections.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+aliases: ['web-servers/nginx/configuration/ssl/','security/ssl/ssl-certificates-with-nginx/','security/ssl/how-to-provide-encrypted-access-to-resources-using-ssl-certificated-on-nginx/','security/ssl/provide-encrypted-resource-access-using-ssl-certificates-on-nginx/']
+modified: 2018-01-09
+modified_by:
+  name: Linode
+published: 2018-01-09
+title: 'Getting Started with NGINX - Part 3: Enable TLS for HTTPS Connections'
+---
+
+![HTTPS Configuration on NGINX](/docs/assets/nginx-ssl/Enable_SSL_nginx.jpg)
```
Comment (EdwardAngert) [inline docs/web-servers/nginx/enable-tls-on-nginx-for-https-connections.md]: @cwlinode should this graphic be updated to match the title? currently reads "Enable SSL for HTTPS  Configuration on NGiNX"

Comment (cwlinode) [reply docs/web-servers/nginx/enable-tls-on-nginx-for-https-connections.md]: Actually, if you could please remove that line so the graphic doesn't show, I'll update that image and add add'l ones after publish.

TIA.

**Comment by GuessWhoSamFoo** [comment]
@EdwardAngert Can you list the relevant files of this PR?

**Comment by cwlinode** [comment]
Closed for https://github.com/linode/docs/pull/1509.

**Pull request closed without merging**